scripts/download_wasm.py

```diff
@@ -68,7 +72,8 @@ def download_wasm(version: str) -> bool:
         print(f'Cannot get cloudproof_rust {version} ({exception})')
         return False
 
+
 if __name__ == '__main__':
     RET = download_wasm('v2.1.0')
     if RET is False and getenv('GITHUB_ACTIONS'):
-        download_wasm('last_build/add_logs_on_findex_callbacks')
+        download_wasm('last_build/update_cover_crypt_12')
```

Comment:
why is it named _12?

Reply:
it is the git branch name (for covercrypt 12.0). While it is not released yet, CI use this cloudproof_rust build to test JS code